frontend/simple/components/CreateGroup/CreateGroupRules.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div id="rulesStep" class="has-text-centered">
+    <h1 class="title is-2"><i18n>Group Advanced Rules</i18n></h1>
+    <p class="content"><i18n>What percentage approval is necessary to adjust the group rules?</i18n></p>
+    <div class="columns">
+      <div class="column">
+        <div class="field" v-if="form.editChange">
+          <div class="control">
+            <input
+              class="input is-large"
+              type="number"
+              min="1"
+              max="100"
+              name="changePercentage"
+              :value="group.changePercentage"
+              @input="update"
+            />
+          </div>
+        </div>
+        <p class="percent" v-else>{{ group.changePercentage }}%</p>
+        <p class="title is-5"><i18n>Change Rules</i18n></p>
+        <button id="changeRulesToggle" class="button is-link is-6 subtitle" @click="toggle('editChange')"><i18n>Change</i18n></button>
+      </div>
+      <div class="column">
+        <div class="field" v-if="form.editApprove">
+          <div class="control">
+            <input
+              class="input is-large"
+              type="number"
+              min="1"
+              max="100"
+              name="memberApprovalPercentage"
+              :value="group.memberApprovalPercentage"
+              @input="update"
+            />
+          </div>
+        </div>
+        <p class="percent" v-else>{{ group.memberApprovalPercentage }}%</p>
+        <p class="title is-5"><i18n>Add Member</i18n></p>
+        <button id="approveToggle" class="button is-link is-6 subtitle" @click="toggle('editApprove')"><i18n>Change</i18n></button>
+      </div>
+      <div class="column">
+        <div class="field" v-if="form.editRemove">
+          <div class="control">
+            <input
+              class="input is-large"
+              type="number"
+              min="1"
+              max="100"
+              name="memberRemovalPercentage"
+              :value="group.memberRemovalPercentage"
+              @input="update"
+            />
+          </div>
```

Comment:
What do you think about this: instead of having the "change" link turn the textfield into an input field, have it reveal a slider below the numbers that the user can click and drag to adjust the number?

Reply:
Also: I'm reminded by this quote from our post on *[Deprecating Majority Rule](https://groupincome.org/2016/09/deprecating-mays-theorem/):*

> Therefore, it is dangerous to use majority rule whenever supermajority rule should be used instead, e.g. passing laws. That is why we plan on making supermajority the default threshold for removing members in Group Income, since doing so can have a potentially significant impact on a person’s life.

We should make it so that a function is called when the slider is dragged, and when it's dragged to a "bad place", Group Income will be able to make a warning field appear that explains to them that they're choosing a value that is most likely too low, and even linking them to our blog posts, etc.